server/container_start.go

```diff
@@ -30,13 +30,19 @@ func (s *Server) StartContainer(ctx context.Context, req *types.StartContainerRe
 		// into the restore code.
 		log.Debugf(ctx, "Restoring container %q", req.ContainerId)
 
+		// Using someNameOfTheContainersImage as a file looks quite suspicious. Luckily(?), AFAICS, as of 2023-10, the TargetFile
+		// field is not used by ContainerRestore at all, so it doesn’t matter.
```

Comment:
I don’t understand what is really going on here…

Reply:
… anyway, I have outright removed that field value. It clearly doesn’t do anything, so let’s not even pretend to.